packages/hw-transport-node-hid-noevents/src/TransportNodeHid.js

```diff
@@ -25,8 +29,20 @@ export function getDevices(): Array<*> {
   return HID.devices(ledgerUSBVendorId, 0x0).filter(filterInterface);
 }
 
-const isDisconnectedError = (e) =>
-  e && e.message && e.message.indexOf("HID") >= 0;
+// Mapping errors from https://github.com/node-hid/node-hid/blob/master/src/HID.cc
+// verbatim inconsistent casing.
+const mapError = (e) => {
+  if (e && e.message) {
+    switch (e.message) {
+      case "Cannot write to closed device":
+      case "could not read from HID device":
+        return new DisconnectedDevice(e);
+      case "Cannot write to hid device":
+        return new CantWriteDevice(e);
```

Comment:
I have feeling that we don't need these two error at all and could stick to the

DisconnectedDeviceDuringOperation

Which is exactly that paradigm of "unexpected interruption" which is the case for many transport implementation too.

Reply:
You probably know more about this than I do, the remapping was merely to cover the "Cannot write to hid device" error message from reaching the user

Reply:
essentially i suggest we remap it all to DisconnectedDeviceDuringOperation
let's check if this idea is acceptible regarding the wording it produces but imo all "DisconnectedDeviceDuringOperation" means "you interrupted something when we were trying to exchange with the device"

Reply:
Do we have a way to reproduce the error consistently?

Reply:
i guess unplug during an apdu exchange, but not sure. probably another way to glitch usb.